dhcpv4/server4/conn_unix.go

```diff
@@ -1,3 +1,5 @@
+// +build !windows
```

Comment:
are build tags still needed after renaming the file?

Reply:
Yes otherwise for Windows, it says:

```
$ GOOS=windows go build .
# github.com/insomniacslk/dhcp/dhcpv4/server4
./conn_unix.go:18:13: undefined: unix.Socket
./conn_unix.go:18:25: undefined: unix.AF_INET
./conn_unix.go:18:39: undefined: unix.SOCK_DGRAM
./conn_unix.go:18:56: undefined: unix.IPPROTO_UDP
./conn_unix.go:27:12: undefined: unix.SetsockoptInt
./conn_unix.go:27:35: undefined: unix.SOL_SOCKET
./conn_unix.go:27:52: undefined: unix.SO_BROADCAST
./conn_unix.go:31:12: undefined: unix.SetsockoptInt
./conn_unix.go:31:35: undefined: unix.SOL_SOCKET
./conn_windows.go:9:6: NewIPv4UDPConn redeclared in this block
	previous declaration at ./conn_unix.go:17:69
```